lesson_07/homewors_Den_07.py

```diff
@@ -0,0 +1,172 @@
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while multiplier <= 5: #Хоть пасхалка про ошибку на 15 строке, я все же решил поменять условие цикла
+        result = number * multiplier
+        # десь тут помила, а може не одна
+        if  result > 25: #Инт сравнить с строкой нельзя
+            # Enter the action to take if the result is greater than 25
+            pass
+        print(str(number) + "x" + str(multiplier) + "=" + str(result))
+
+        # Increment the appropriate variable
+        multiplier += 1
+#ПО итогу получаю требуемый вывод
+multiplication_table(3)
+# Should print:
+# 3x1=3
+# 3x2=6
+# 3x3=9
+# 3x4=12
+# 3x5=15
+
+# task 2
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+
+def sum_of_two_numbers(first, second):
+    result = 0
+    if (type(first) == int or type(first) == float) and (type(second) == int or type(second) == float):
+        result = first + second
+        print(f"{first}+{second}={result}")
```

Comment:
```suggestion
        retunrn result
```
ну ми ж уже в першому класі, тому повертаємо значення